dlc/src/main/scala/org/bitcoins/dlc/testgen/DLCTestVector.scala

```diff
@@ -109,15 +111,18 @@ object ValidTestInputs {
   }
 }
 
-// TODO: Add realOutcome, oracleSignature, offerSignedCET, acceptSignedCET fields
+case class DLCTransactions(
```

Comment:
Why not use `SetupDLC` here?

Reply:
a) The extra adaptor signatures are redundant
b) simpler
c) wanted something I could freely change within the test vector code that wasn't mingled with other external stuff, I don't view this as particularly stable but if in the future it truly doesn't change I'd be open to switching to something like `SetupDLC` and using outcomes and such. Though I will note that part of what's unstable is I haven't generated negative test vectors yet and those may end up changing this to contain options and such